### PR TITLE
Issue #2946464 text capitalisation

### PR DESCRIPTION
**Pull request by left23**
## Problem
Capitalization in titles gives problems, capital letters are harder to read

## Solution
In SocialBase and SocialBlue, I commented out all uppercase text declarations. I did not delete for now, as font-size adjustment might be needed and we can easily find these with a text search.

## Issue tracker
https://www.drupal.org/project/social/issues/2946464

## HTT
- [ ] Login as LU+ and look at the stream
- [ ] Notice which components' text is uppercase (refer to screenshots on issue)
- [ ] Checkout to this branch
- [ ] Notice how the text is no longer uppercase

## Documentation
- [ ] This item is added to the release notes
- [ ] This item is documented on LGOS
- [ ] This item has been added to the feature overview

### CODE REVIEW

**Comment by left23** [comment]
@Crazylord I am waiting for Bas to review a branch I've deployed with this PR and https://github.com/goalgorilla/open_social/pull/808
He might suggest changes etc so I just wanted you to be aware of this PR.

**Comment by Crazylord** [comment]
Hi @basvanos can you check this.

File: themes/socialbase/components/01-base/tables/_tables.scss [hunk 1]
```diff
@@ -48,7 +48,7 @@ th {
 
   // Bottom align for column headings
   thead th {
-    text-transform: uppercase;
+    //text-transform: uppercase;
```
Comment (jaapjan) [inline themes/socialbase/components/01-base/tables/_tables.scss]: @Crazylord can you decide this? I think we should just remove this line? Having commented css is not that nice.

File: themes/socialbase/components/02-atoms/cards/cards.css [hunk 2]
```diff
@@ -0,0 +1,61 @@
+/*
```
Comment (jaapjan) [inline themes/socialbase/components/02-atoms/cards/cards.css]: @Crazylord  this does not look good. Can you verify whats happening and re-generate?

Comment (Crazylord) [reply themes/socialbase/components/02-atoms/cards/cards.css]: Fixed problem with this. Re-generated css files.

File: themes/socialbase/components/01-base/tables/_tables.scss (hunk 1 above)
Comment (Crazylord) [reply themes/socialbase/components/01-base/tables/_tables.scss]: Removed this.

**Comment by bramtenhove** [comment]
As this still requires work I removed the 2.0 milestone. Please let me know if this is incorrect and I'll revert it.

**Comment by Crazylord** [comment]
Hi @bramtenhove . Look like there all works si done.

File: themes/socialbase/components/02-atoms/cards/cards.css (hunk 2 above)
Comment (jaapjan) [reply themes/socialbase/components/02-atoms/cards/cards.css]: @Crazylord this file still looks broken:
themes/socialbase/components/02-atoms/cards/cards.css which is why the PR is on "needs work"

**Pull request closed without merging**